app/controllers/catalog_controller_decorator.rb

```diff
@@ -57,11 +57,13 @@
   # later load the dog biscuits translations.
   HykuKnapsack::Engine.load_translations!
 
+  # @todo remove this and list index properties individually
   index_props = DogBiscuits.config.index_properties.collect do |prop|
     { prop => CatalogController.send(:index_options, prop, DogBiscuits.config.property_mappings[prop]) }
   end
   CatalogController.send(:add_index_field, config, index_props)
   config.add_index_field 'all_text_tsimv', label: "Item contents", highlight: true, helper_method: :render_ocr_snippets, if: :query_present?
+  config.add_index_field 'based_near_label_tesim', itemprop: 'contentLocation', link_to_facet: 'based_near_label_sim'
```

Comment:
This fixes the form so you don't have to hit + to open the location before searching